### PR TITLE
Adding a sanity check to ensure we're sending tokens to the correct place

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: INT-154

Added because a user tried to transfer sUSD through the bridge, saw it didn't work, and then modified this sample code to do it.

### CODE REVIEW

File: l1-l2-deposit-withdrawal/scripts/example.js [hunk 1]
```diff
@@ -63,16 +63,21 @@ async function main() {
     'L1 ERC20', //name
   )
   await L1_ERC20.deployTransaction.wait()
+  console.log(`   @addr ${L1_ERC20.address}`)
 
   // Deploy the paired ERC20 token to L2.
   console.log('Deploying L2 ERC20...')
   const L2_ERC20 = await factory__L2_ERC20.connect(l2Wallet).deploy(
     '0x4200000000000000000000000000000000000010',
+    // The first value is to check the case of an invalid L1. The second value
```
Comment (elenadimitrova) [inline l1-l2-deposit-withdrawal/scripts/example.js:72]: I don't understand this 3 line-comment ?

File: l1-l2-deposit-withdrawal/scripts/example.js [hunk 2]
```diff
@@ -63,16 +63,21 @@ async function main() {
     'L1 ERC20', //name
   )
   await L1_ERC20.deployTransaction.wait()
+  console.log(`   @addr ${L1_ERC20.address}`)
```
Comment (elenadimitrova) [inline l1-l2-deposit-withdrawal/scripts/example.js:66]: The added console.logs are just outputting the addresses but maybe they should be prefixed with some wording on what address if printed, e.g. "L1_ERC20 address" and "L2_ERC20 address" instead of just "0x..." outputs

File: l1-l2-deposit-withdrawal/scripts/example.js [hunk 3]
```diff
@@ -91,6 +96,17 @@ async function main() {
   const tx1 = await L1_ERC20.approve(L1StandardBridge.address, 1234)
   await tx1.wait()
 
+  // Sanity check, is the L2 address really the correct IL2StandardERC20 for this
+  // L1 token. If the contract doesn't implement IL2StandardERC20 we'll have an exception
+  // and the transfer will also fail.
+  //
+  // DO NOT remove this test. Without it a transfer might get blocked, and you won't get the
+  // assets back until seven days have passed. It might even be possible assets will be lost.
```
Comment (elenadimitrova) [inline l1-l2-deposit-withdrawal/scripts/example.js:104]: This wording doesn't sound good. Maybe reword to:
  
"DO NOT remove this check.
Ensures L2 token compliance and validity. If the L2 token contract doesn't implement `IL2StandardERC20` or it does not correspond to the L1 token being deposited, an exception will occur and no deposit will take place. 
Alternatively the exception will occur once he deposit is relayed to L2 and the seven day wait period will apply for the bad deposit to be withdraw-able back on L1"

File: l1-l2-deposit-withdrawal/scripts/example.js (hunk 1 above)
Comment (qbzzt) [reply l1-l2-deposit-withdrawal/scripts/example.js]: Sorry, clarified.

File: l1-l2-deposit-withdrawal/scripts/example.js [hunk 4]
```diff
@@ -91,6 +96,21 @@ async function main() {
   const tx1 = await L1_ERC20.approve(L1StandardBridge.address, 1234)
   await tx1.wait()
 
+  // Sanity check, is the L2 address really the correct IL2StandardERC20 for this
+  // L1 token. If the contract doesn't implement IL2StandardERC20 we'll have an exception
+  // and the transfer will also fail.
```
Comment (elenadimitrova) [inline l1-l2-deposit-withdrawal/scripts/example.js:101]: Let's merge these 3 lines with what's below as they overlap
>is the L2 address really the correct IL2StandardERC20 for this L1 token. If the contract doesn't implement IL2StandardERC20..

is same as 

>If the L2 token contract doesn't implement IL2StandardERC20 or it does not correspond to the L1 token being deposited...

**Review by elenadimitrova — APPROVED** [review]
